lib/ash/resource/validation/action_is.ex

```diff
@@ -22,6 +24,14 @@ defmodule Ash.Resource.Validation.ActionIs do
 
   @impl true
   def describe(opts) do
-    [message: "must be %{action}", vars: %{action: opts[:action]}]
+    {operator, action} =
+      case opts[:action] do
+        [action] -> {"==", action}
+        actions -> {"in", actions}
+      end
+
+    [
+      message: "must be #{operator} #{inspect(action)}"
+    ]
```

Comment:
for backwards compatibility, lets retain the old behavior when `opts[:action]` has just one item. For the new behavior, lets still use `vars`, something like

`[message: "action must be one of %{actions}", vars: [actions: Enum.join(actions, ", ")]`